apps/src/cookieBanner/cookieBanner.js

```diff
@@ -0,0 +1,27 @@
+import cookies from 'js-cookie';
+import { getRootDomainFromHostname } from '@cdo/apps/code-studio/utils';
+
+window.setupCookieBanner = (environment) => {
+  const cookieName = '_cookieBanner' +
```

Comment:
Two questions:
- Why do we put leading underscores on our cookie names?
- Why don't we 'namespace' our cookie names? e.g. `cdo_cookieBanner`

Reply:
I suspect this goes way back to when our original `language` cookie got superseded by a `_language` cookie, though I don't know why that particular naming choice was named.  A leading underscore is now internally inconsistent, if not particularly nice looking.  I guess we could start namespacing cookies but then only new cookies would get that.